modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json

```diff
@@ -0,0 +1,114 @@
+{
+	"tailored_jacket": [
+		{
+			"type": "icon_state",
+			"icon_state": "longcoat_base_obj",
```

Comment:
Out of curiosity is there a reason for all the component states being longcoat with the full icon being tailored_jacket? I'd hate to say you **need** to keep them consistent so don't worry, just something to keep in mind in future if anything

Reply:
the answer is i didn’t know what i was going to call this thing at this point